.gitconfig

```diff
@@ -0,0 +1,2 @@
+[safe]
```

Comment:
Not sure I understand why this is needed

Reply:
Hmm... no idea where this file is coming from. It isn't needed as far as I know. Deleting it.